<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manager</class>
 <widget class="QMainWindow" name="manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget_manager">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_managementPersonnel">
       <attribute name="title">
        <string>Management of personnel</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_listPersonnel">
            <property name="text">
             <string>List of personnel :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="tableView"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_name">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_firstName">
              <item>
               <widget class="QLabel" name="label_firstName">
                <property name="text">
                 <string>First name :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_firstName"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_lastName">
              <item>
               <widget class="QLabel" name="label_lastName">
                <property name="text">
                 <string>lastName :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_LastName"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_birthDate">
            <item>
             <widget class="QLabel" name="label_birthDate">
              <property name="text">
               <string>Birth date (DD/mm/yyyy) :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_birthDate"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_personnel">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Personnel :</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_radioButton">
              <item>
               <widget class="QRadioButton" name="radioButton_controller">
                <property name="focusPolicy">
                 <enum>Qt::StrongFocus</enum>
                </property>
                <property name="acceptDrops">
                 <bool>false</bool>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Controller</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_manager">
                <property name="text">
                 <string>Manager</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_address">
            <item>
             <widget class="QLabel" name="label_iban">
              <property name="text">
               <string>IBAN :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_iban"/>
            </item>
            <item>
             <widget class="QLabel" name="label_">
              <property name="text">
               <string>Address :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_address"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_city">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_city">
              <item>
               <widget class="QLabel" name="label_city">
                <property name="text">
                 <string>City :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_city"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_postalCode">
              <item>
               <widget class="QLabel" name="label_postalCode">
                <property name="text">
                 <string>Postal code :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_postalCode"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_Email">
            <item>
             <widget class="QLabel" name="label_email">
              <property name="text">
               <string>Email :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_email"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_phone">
            <item>
             <widget class="QLabel" name="label_phone">
              <property name="text">
               <string>Phone :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_phone"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_action">
            <item>
             <widget class="QPushButton" name="pushButton_add">
              <property name="text">
               <string>&amp;Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_modify">
              <property name="text">
               <string>&amp;Modify</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_delette">
              <property name="text">
               <string>&amp;Delette</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_resetPassword">
            <property name="text">
             <string>Reset password</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_statistics">
       <attribute name="title">
        <string>Statistics</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
